spec/services/page_service_spec.rb

```diff
@@ -14,13 +14,36 @@
       expect(subject.list).to match_array([en_page, fr_page])
     end
 
-    it 'limits result to a 100' do
-      expect_any_instance_of(ActiveRecord::QueryMethods).to receive(:limit).with(100){ Page.all }
+    it 'limits result to 30 by default' do
+      expect_any_instance_of(ActiveRecord::QueryMethods).to receive(:limit).with(30){ Page.all }
       subject.list
     end
 
+    it 'limits result by passed value' do
+      expect(subject.list(limit:1)).to match([fr_page])
```

Comment:
Is it simpler to test only length? (that's what concerns `limit`). E.g. if we decide to change it to: `.order('created_at asc')` or custom sort field, this test would need to be updated even though it's unrelated.